ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -313,6 +313,31 @@ defaults <- node_group <- node_group.node <- node
     igmp_snooping_enabled: < true | false >
 ```
 
+### BGP & EVPN Multi-Domain Gateway
+```yaml
+< node_type_key >:
+
+  defaults:
+    # Node is acting as EVPN Multi-Domain Gateway | Optional.
+    # New BGP peer-group is generated between EVPN GWs in different domains or between GWs and Route Servers. Name can be changed under "bgp_peer_groups.evpn_overlay_core" variable
+    # L3 rechability for different EVPN GWs must be already in place, it is recommended to use DCI & L3 Edge if Route Servers and GWs are not defined under the same Ansible inventory.
+    evpn_gateway:
+      remote_peers_vxlan: [ '< inventory_hostname_of_remote_evpn_gw_server >' ]
+
+      # When remote EVPN GWs or Route Servers are not present in the running Ansible inventory, they can be defined statically.
+      remote_peers_vxlan_neighbors:
```

Comment:
Try to collapse the data models, so we just have a single list of peers. Then we can have a key for hostname, and if the hostname is in the fabric, we will pick can pick up the ip and AS automatically, unless they are set (so allow to override)
If you call it `remote_peers`, we can later add the mpls relevant keys under the same list, but only require them with mpls gateway.